src/utils/fluid_stub_functions.h

```diff
@@ -31,30 +31,30 @@
 #include "fluidsynth_priv.h"
 
 
-#define STUB_FUNCTION_VOID(function, args...) \
+#define STUB_FUNCTION_VOID(function, ...) \
     static FLUID_INLINE void \
-    function(args) \
+    function(__VA_ARGS__) \
```

Comment:
I think using `__VA_ARGS__` would require to bump `CMAKE_C_STANDARD` to `99`? I'm unsure if older version of MSVC would buy that.